roles/sap_ha_install_anydb_ibmdb2/README.md

```diff
@@ -0,0 +1,65 @@
+`EXPERIMENTAL`
+
+# sap_ha_install_anydb_ibmdb2 Ansible Role
+
+Ansible Role for instantiation of IBM Db2 “Integrated Linux Pacemaker” HADR cluster
```

Comment:
Instantiation is common technical term for creation of an instance, it's in various documents from different vendors. Reason word is chosen is because it's not really "creating", it's "turning on" the feature.